source/unified-test-format/unified-test-format.rst

```diff
@@ -3893,7 +3941,7 @@ Changelog
 
 ..
   Please note schema version bumps in changelog entries where applicable.
-
+:2023-05-10: Add runCursorCommand and createRunCursorCommand operations
```

Comment:
This should be amended with "Added a CommandCursor entity type", like the previous changelog entry for schema version 1.3.

Reply:
updated